include/deal.II/distributed/tria_base.h

```diff
@@ -83,6 +83,9 @@ namespace parallel
     const std::vector<unsigned int> &
     n_locally_owned_active_cells_per_processor() const;
 
+    void
+    update_n_locally_owned_active_cells_per_processor();
```

Comment:
Please add a comment and mention in the previous function that this function needs to be called to make the data available.